taxcalc/dropq/dropq.py

```diff
@@ -1,6 +1,7 @@
 """
 The dropq functions are used by TaxBrain to call Tax-Calculator in order
-to maintain the privacy of the micro data being used by TaxBrain.
+to maintain the privacy of the micro data being used by TaxBrain.  This
+is done by adding random "fuzz" to the results in each table cell.
```

Comment:
Possibly: "This is done by adding random "fuzz" to the sample from which the results in each table cell are drawn."

Reply:
You're right.  The description of what "fuzzing" means needs to be improved.
Commit 537b1c0 is an attempt to improve the documentation.